avaframe/ana1Tests/influenceTest.py

```diff
@@ -0,0 +1,297 @@
+    param2Study: string
+        main parameter under study
+    simDF: Data-Frame
+        contains results of DFA simulations
+    """
+
+    # create one cfg files for each releaseScenarios and fetch input data
+    inputSimFilesAll, cfgFilesRels = com1DFA.setRelThIni(avalancheDir, com1DFA, cfgFile=cfg)
+    # loop over cfg files of all release scenarios
+    IDList = []
+    for cfgFileRel in cfgFilesRels:
+        log.debug('Full cfg file %s' % cfgFileRel)
+        # copy inputSimFilesAll - as this becomes changed according to current release scenario
+        inputSimFiles = inputSimFilesAll.copy()
+        # reset variationDict
+        variationDict = ''
+        # get information on simulations that shall be performed according to parameter variation
+        modCfg, variationDict = dP.getParameterVariationInfo(avalancheDir, com1DFA, cfgFileRel)
+        # select release input data according to chosen release scenario
+        inputSimFiles = gI.selectReleaseScenario(inputSimFiles, modCfg['INPUT'])
+        # TODO: once it is confirmed that inputSimFiles is not changed within sim
+        # keep for now for testing
+        inputSimFilesTest = inputSimFiles.copy()
+        # create a list of simulations and generate an individual configuration object for each simulation
+        # if need to reproduce exactly the hash - need to be strings with exactely the same number of digits!!
+        # first get info on already existing simulations in Outputs
+        simDFOld, simNameOld = cfgUtils.readAllConfigurationInfo(avalancheDir, specDir='')
+        # prepare simulations to run (only the new ones)
+        IDListi, _ = com1DFA.prepareVarSimDict(modCfg, inputSimFiles, variationDict, simNameOld=simNameOld)
+        for ID in IDListi:
+            IDList.append(ID)
+
+    return IDList
+
+
+
+def generateAutoIniFile(cfg, viscName):
+    """ generates .ini files starting from ana1Tests/influenceTestCfg.ini and combining datas with the section
+    related to the viscosity to study.
+    Parameters
+    ----------
+    cfg: path to an .ini file
+    viscName: string
+        has to be 'ata' or 'samos'
+    """
+    log.info("Write " + viscName + " simulation settings in '" + viscName + "Cfg.ini' file")
+    config.read(cfg)
+    paramNameList = []
+    paramValueList = []
+    for param in config[viscName]:
+        print('param = ', param)
+        paramNameList.append(param)
+        paramValueList.append(config[viscName][param])
+
+    if viscName == 'samos':
+        paramNameList.append('viscOption')
+        paramValueList.append('1')
+    elif viscName == 'ata':
+        paramNameList.append('viscOption')
+        paramValueList.append('2')
+
+    simTypeList = config['GENERAL']['simTypeList']
+    if simTypeList != 'null':
+        log.info('Please modify source code yourself to distinguish simulations with entrainment or not')
+        log.info('It will be an option developped late sorry')
```

Comment:
Leave this out and write
in the next log.info: For now simTypeList has to be null